src/popup.js

```diff
@@ -0,0 +1,25 @@
+const displayData = document.querySelector('body > main > section');
+export default function commentPopup(image, title, text, id) {
+  const commentDiv = document.createElement('div');
+  commentDiv.classList.add('main-popup');
+  commentDiv.innerHTML = `
+        <span id="close-btn"><i class="fal fa-times"></i></span>
+        <div class="popup-image">
+           <img src="${image}" class="img" alt="close-icon">
+        </div>
```

Comment:
- [ ] After you fix the id for your comments functionality. You should add styles to your popup image because it has a huge size.
- [ ] The images you are showing in your popup don't belong to the right item on your home page. Please fix them too. ♻️